### ISSUE TITLE
Releasing the annotated  images

### ISSUE THREAD

**Issue by rafiibnsultan** [VIDA-NYU/tile2net#26]
Hello,
Do you intend to release the dataset that you used to train your model?
"To achieve that, our research team manually corrected 2,500 tiles of the 12,000 training set, 1,620 image tiles out of 4,000 tiles that were used as our validation set, and 1,500 tiles out of 4,000 test set tiles."

I was thinking will you release these mask images alongside the tiles?

**Comment by Mary-h86** [comment]
Hi,
Yes, we are planning on releasing the data, and we are adding more classes to the model, so new changes are coming! 
We will post the news in the updates section as soon as the data is made publicly available.
